fastlane/Fastfile

```diff
@@ -0,0 +1,72 @@
+system("cd .. && npm install shelljs@0.7.0")
+system("cd .. && ./install.sh")
+
+lane :All do
+gradle(
+task:":libs:SalesforceAnalytics:connectedAndroidTest :libs:SalesforceSDK:connectedAndroidTest :libs:SmartStore:connectedAndroidTest :libs:SmartSync:connectedAndroidTest :libs:SalesforceHybrid:connectedAndroidTest :native:NativeSampleApps:RestExplorer:connectedAndroidTest",
+flags: "--continue --no-daemon --profile --stacktrace"
+)
+end
+
+lane :PR do
+gradle(
+task:":libs:SalesforceAnalytics:connectedAndroidTest :libs:SmartStore:connectedAndroidTest :libs:SmartSync:connectedAndroidTest :libs:SalesforceHybrid:connectedAndroidTest",
```

Comment:
Since our PR is running on android-23 version that is most reliable version we recognized so far, for which the emulator has an known issue that cannot setup proxy for emulator to use. We only can do that for android-19. Without the proxy server setup, test testRestClientUnauthenticatedlientInfo will consistently failed.